rotkehlchen/data_import/importers/bittrex.py

```diff
@@ -24,14 +23,27 @@
 if TYPE_CHECKING:
     from collections.abc import Callable
 
-logger = logging.getLogger(__name__)
-log = RotkehlchenLogsAdapter(logger)
+
+class BittrexFileType(Enum):
+    """
+    The type of the file we are parsing
+    """
```

Comment:
if it fits in one line keep it short. Also maybe
```suggestion
    """Different type of CSV files provided by bittrex"""
```